src/logstash/scripts/facilities/facility_type.rb

```diff
@@ -1,16 +1,16 @@
 def filter(event)
-  # Get the facility_type_value (assuming it's a field in the event)
+  # Get the facility_type_value (assuming it's a field in the event).
```

Comment:
Do we really need those changes here and in other places? I mean adding dots at the end of the comment line.